tests/test_async_client.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+from unittest.mock import AsyncMock, patch
+import unittest.async_case
+import willofsteel
+from willofsteel.async_client import AsyncClient
+from willofsteel.exceptions import *
+from willofsteel.types import *
+
+
+class TestAsyncClient(unittest.async_case.IsolatedAsyncioTestCase):
+
+    def setUp(self):
+        patcher = patch(
+            'willofsteel.async_client.AsyncClient.request', new_callable=AsyncMock)
+        self.addCleanup(patcher.stop)
+        self.mock_request = patcher.start()
+        self.mock_request.return_value = {}, 200
+        self.client = AsyncClient('valid_key')
```

Comment:
```suggestion
        self.client = AsyncClient('valid_key', logger=LoggingObject(level=logging.CRITICAL))
```

Logs from the application itself shouldn't show unless it's a error.